tests/unit/test_xaprintf.c

```diff
@@ -76,15 +78,15 @@ test_xaprintf_exit(void **state)
 		assert_unreachable();
 		break;
 	case EXIT_CALLED:
-		assert_true(strcmp(p, "xaprintf_called"));
+		assert_true(streq(p, "xaprintf_called"));
```

Comment:
There is already assert in cmocka to compare strings
`assert_string_equal`

Reply:
Yeah, but I prefer chaining calls.  I don't see the value of that API.  It's simpler to use simpler APIs.

Plus, streq() will hopefully be standard C rather soon (two months).